src/Model/Event.php

```diff
@@ -344,7 +344,6 @@ public static function getItemArrayForId(int $event_id, array $item = []): array
 		$event['id'] = $event_id;
 
 		$item['uid']           = $event['uid'];
-		$item['contact-id']    = $event['cid'];
```

Comment:
Why is it removed?

Reply:
Array key is duplicated in the same block.